subsys/bluetooth/shell/bt.c

```diff
@@ -1424,9 +1501,74 @@ static int cmd_adv_param(const struct shell *sh, size_t argc, char *argv[])
 	return 0;
 }
 
+static const uint8_t ad_ext_uuid16[] = {
+	IF_ENABLED(CONFIG_BT_ASCS, (BT_UUID_16_ENCODE(BT_UUID_ASCS_VAL),))
+	IF_ENABLED(CONFIG_BT_BASS, (BT_UUID_16_ENCODE(BT_UUID_BASS_VAL),))
+	IF_ENABLED(CONFIG_BT_PACS, (BT_UUID_16_ENCODE(BT_UUID_PACS_VAL),))
+	IF_ENABLED(CONFIG_BT_CAP_ACCEPTOR, (BT_UUID_16_ENCODE(BT_UUID_CAS_VAL),))
+	IF_ENABLED(CONFIG_BT_HAS, (BT_UUID_16_ENCODE(BT_UUID_HAS_VAL),))
+	IF_ENABLED(CONFIG_BT_VCS, (BT_UUID_16_ENCODE(BT_UUID_VCS_VAL),))
```

Comment:
I'm curious as to why this specific subset of services were chosen to have their UUID advertised. Can you elaborate?

Reply:
As written in specifications e.g:
BAP:
![image](https://user-images.githubusercontent.com/10126877/181756452-e77986e3-687f-49f4-a20b-b47046138ef4.png)
HAP:
![image](https://user-images.githubusercontent.com/10126877/181756614-45b7c1a8-2d6e-43f5-b127-90718c175ae2.png)
VCP:
![image](https://user-images.githubusercontent.com/10126877/181758498-0944106e-09ff-4224-9f3f-c9f08d5f6163.png)
MICP:
![image](https://user-images.githubusercontent.com/10126877/181758983-d56e7472-926b-4438-ae91-ffc745115837.png)

Reply:
I get that, but I was wondering why this specific **subset** of the services were chosen, because e.g. CCP has similar text but the TBS service UUID isn't added here. I assume that MICP also includes similar language.

Reply:
I will extend the data with missing services. I just forgot about those, that's why

Reply:
Alright :D